tech-ally/src/app/models/commit.ts

```diff
@@ -0,0 +1,45 @@
+import { CommitDto } from "@app/dtos/commit.dto";
+import { IssueDto } from "@app/dtos/issue.dto";
+import "@app/extensions/date";
+
+export class Commit {
+    name: string;
+    author: string;
+    url: string;
+    created: string;
+    sha!: string;
+    type!: string;
+
+    constructor(name: string, author: string, url: string, created: string, sha: string, type: string) {
+        this.name = name;
+        this.author = author;
+        this.url = url;
+        this.created = created;
+        this.sha = sha;
+        this.type = type;
+    }
+
+    static fromDto(commitDto: CommitDto): Commit{
+        let message = commitDto.commit.message.split(":")
```

Comment:
I know all our commits will have this character but, should we add protection? what will happen if the commit doesn't have it and you access `message[1]` below?

Reply:
Yeh, good point i'll add some protection around this. 👍